scripts/src/main/resources/scripts/command/aws

```diff
@@ -71,45 +67,26 @@ function doPackageInstall()
 }
 
 function doSetup() {
-  local ERR=""
-  findAws
-  if [[ ${AWS_INSTALLED} != "yes" ]] || [[ "${1}" != "silent" ]]
+  local silent="${1}"
+  doInstall "aws" "${AWS_VERSION}" "${silent}" "" "${DEVON_IDE_HOME}/updates/install/aws"
+  if [ "${?}" = 0 ]
   then
-    # Get leatest release
-    if [ -z "${AWS_VERSION}" ]
-    then
-      doEcho "Getting latest release..."
-      AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
-    fi
-
-    doInstall "-" "${DEVON_IDE_HOME}/updates/install/aws" "aws" "${AWS_VERSION}"
-    ERR="${?}"
-    if [ "${ERR}" = "0" ]
-    then
-      doPackageInstall "${DEVON_IDE_HOME}/updates/install/aws" "${AWS_VERSION}"
-    fi
-    doRunCommand "rm -rf ${DEVON_IDE_HOME}/updates/install/aws"
-
-    if [ "${1}" != "silent" ] && ! doIsQuiet
-    then
-      doEcho "logon again to test aws with \"aws --version\""
-    fi
+    doPackageInstall "${DEVON_IDE_HOME}/updates/install/aws"
   fi
 }
 
-
 # CLI
 case ${1} in
 "help" | "-h")
   echo "Setup or run AWS CLI (command-line interface for Amazon-Web-Services)."
   echo
   echo "Arguments:"
   echo " setup                          install aws on your machine."
-  echo " �args�                         call aws with the specified arguments. Call aws --help for details or use aws directly as preferred."
+  echo " «args»                         call aws with the specified arguments. Call aws --help for details or use aws directly as preferred."
```

Comment:
where does this come from?
I will fix it...

Reply:
I do not have this locally and can therefore not fix it. Looks like some UTF-8 vs. ISO-Latin encoding conversion bug...

Reply:
For the record: When I got to the tab `Conversation` of this PR then the diff looks correct and nothing has changed about `«args»`.
However, in the tab `Files changed`, I see it changed to `Â«argsÂ»`.
I was referring the these broken `Â` characters that I do not have locally and that I do not understand.
IMHO this is an encoding bug of github when dealing with special unicode characters...
I hope that if we merge this, we do not get these `Â` characters into our code...

Reply:
The help option has been standardized for all commandlets. My change reversed that, so I copied and pasted the change back in. I used copy and paste because I couldn't type the original change because of the special characters. And what also surprised me is that the change was not displayed as a merge conflict in Github and I was able to fix it there.